crates/sui-types/src/multisig.rs

```diff
@@ -66,8 +46,7 @@ pub struct MultiSig {
     sigs: Vec<CompressedSignature>,
     /// A bitmap that indicates the position of which public key the signature should be authenticated with.
     #[schemars(with = "Base64")]
-    #[serde_as(as = "SuiBitmap")]
-    bitmap: RoaringBitmap,
+    bitmap: Vec<u8>,
```

Comment:
IIUC, bcs will convert this to length | bitmap[0] | bitmap[1] - Should we use ULEB128? that way it would always be 1-2 bytes

Where do we set it as a bitmap?